integration-tests/integration/createIncident/create-report-for-prisoner-in-other-prison.spec.js

```diff
@@ -57,6 +57,7 @@ context('Creating reports for prisoners in other prisons', () => {
     reportUseOfForcePage.dob().contains('26 December 2000')
     reportUseOfForcePage.nomisId().contains('A1234AC')
     reportUseOfForcePage.offenderImage().should('be.visible')
+    reportUseOfForcePage.logout().click()
```

Comment:
logout of this test at the end fixes subsequent test which fails in UI only - something to do with image still loading from last test whilst new test is running.